tests/integration_test.sh

```
@@ -1278,9 +1278,114 @@ test_enhanced_transaction_methods() {
+        return 1
+    fi
+
+    # Create sample Ethereum transactions (EIP-1559)
+    # These are real EIP-1559 transactions with valid signatures
+    local tx1="02f872018084773594008504a817c80082520894123456789012345678901234567890123456789087038d7ea4c6800080c080a07db446c5f0f87374845fb7388af19b687fb6304664e4b28bdae3d379e01dca7aa04f7f2286f2cd8eb3b960ce374a4700fe232efbfb0bdc61293407ef9fee38d197"
+    local tx2="02f872018084773594018504a817c80082520894123456789012345678901234567890123456789087038d7ea4c6800080c080a07db446c5f0f87374845fb7388af19b687fb6304664e4b28bdae3d379e01dca7aa04f7f2286f2cd8eb3b960ce374a4700fe232efbfb0bdc61293407ef9fee38d198"
+
+    print_status "Submitting bundle with 2 transactions..."
+
+    # Submit bundle using CLI
+    local bundle_output=$(./target/debug/core-lane-node send-bundle \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)